zfsbootmenu/install-helpers.sh

```diff
@@ -118,7 +118,7 @@ create_zbm_profiles() {
   # Create shell profiles for ZBM
 
   cat > "${BUILDROOT}/etc/profile" <<-EOF
-	export PATH=/usr/sbin:/usr/bin:/sbin:/bin
+	export PATH=/zbm/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
This is a bit hacky, but it's the least invasive way to get `zbm-kcl` into release/recovery images and in a path.